fetch_wrapper.ts

```diff
@@ -37,13 +46,53 @@ export function wrapFetch(options?: WrapFetchOptions): typeof fetch {
     }
     interceptedInit.headers.set("cookie", cookieString);
 
-    const response = await fetch(input, interceptedInit);
+    const response = await fetch(input, {
```

Comment:
can we not use spread operator for this and simply set `redirect` on `interceptedInit` directly ?
the reason is spread has a cost, and I prefer to not use it when It's not needed
even if we want to clone user's `init`, in this case which we should have, let's do it once and when we assign the `interceptedInit`
also probably change `interceptedInit` type to `WrappedFetchRequestInit` ?